docs/reference/intro.asciidoc

```diff
@@ -238,12 +238,13 @@ testing with your own data and queries].
 [[disaster-ccr]]
 ==== In case of disaster
 
-For performance reasons, the nodes within a cluster need to be on the same
-network. Balancing shards in a cluster across nodes in different data centers
-simply takes too long. But high-availability architectures demand that you avoid
-putting all of your eggs in one basket. In the event of a major outage in one
-location, servers in another location need to be able to take over. Seamlessly.
-The answer? {ccr-cap} (CCR).
+A cluster's nodes need good, reliable connections to each other. To provide
+better connections, you typically co-locate the nodes in the same data center or
+nearby data centers. Balancing shards across nodes that are far away or not well
+connected simply takes too long. However, to maintain high availability, you
```

Comment:
I think the sentence about balancing shards doesn't really fit here. Basically any request might go between nodes and suffer from latency or disruption, and things like searches are likely a lot more latency-sensitive than shard movements. Maybe just drop it or maybe replace it with something along those lines instead.

```suggestion
nearby data centers. However, to maintain high availability, you
```

Reply:
Great suggestion. Thanks, David!